CMakeLists.txt

```diff
@@ -14,7 +22,7 @@ if (vw_USE_AZURE_FACTORIES)
 endif()
 
 find_package(OpenSSL REQUIRED)
-find_package(Boost COMPONENTS unit_test_framework system program_options thread REQUIRED)
+find_package(Boost COMPONENTS unit_test_framework system program_options thread REQUIRED) # Boost::uuid too, but that cannot be "found" here
```

Comment:
where is it found?

Reply:
It is a header-only library, which cannot be found as a component by find_package()